python/katana/bug/__main__.py

```diff
@@ -0,0 +1,32 @@
+import argparse
+import sys
+from pathlib import Path
+
+from katana.bug import capture_environment
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(
+        prog=f"{Path(sys.executable).name} -m katana.bug",
+        description="""
+        Capture environment information for bug reporting.
+        """,
+    )
+
+    parser.add_argument(
+        "destination",
+        help="Output the binary to the given path or stdout if '-' is given",
+        type=str,
+        nargs="?",
+        default=None,
+    )
+
+    args = parser.parse_args()
+
+    if args.destination == "-":
```

Comment:
This feels a bit odd, i would say make it explicit "stdout".

Reply:
`-` meaning stdout is reasonably common in Unix tools. Which is why I used it. "stdout" isn't great because it's something that could be a file name. I don't want it to be the default because sending to stdout is an expert feature. I'll use an `--stdout` option instead though. The old Unix conventions are not well known any more.

Reply:
ah if it's common than it's fine. I haven't seen it before so lack of knowledge there.